src/main/resources/fleet_mgmt_location.avro

```diff
@@ -0,0 +1,56 @@
+{
+    "namespace": "fleet_mgmt",
+    "name": "fleet_mgmt_location",
+    "type": "record",
+    "fields": [
+        {
+            "name": "vehicle_id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "range": {
+                        "min": 1000,
+                        "max": 9999
+                    }
+                }
+            }
+        },
+        {
+            "name": "latitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -50,
+                        "max": 70
+                    }
+                }
+            }
+        },
+        {
+            "name": "longitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -175,
+                        "max": 175
+                    }
+                }
+            }
+        },
```

Comment:
These field values won't be very useful (beyond looking like coordinates) - most wouldn't make sense for vehicle location so downstream maps / distance calculations wouldn't be realistic. I'd suggest a single `lat_long` `array` with a small set (25? 50?) of  `options` that _do_ make sense -- e.g., grab real coordinates from a a real driving route (can get coordinates from google maps)